doc/user_doc/vic_installation/vch_installer_options.md

```diff
@@ -150,17 +213,26 @@ Use this option in combination with the `cert` option, that provides the path to
 The `vic-machine` utility provides options to customize the deployment of virtual container hosts.
 
 ### `name` ###
+
+Short name: `-n`
+
 A name for the virtual container host appliance. If not specified, `vic-machine` sets the name of the virtual container host to `docker-appliance`. If a virtual container host of the same name exists on the ESXi host or in the vCenter Server inventory, or if a folder of the same name exists in the target datastore, the deployment of the virtual container host fails. Use the `force` option to replace an existing virtual container host with a new one with the same name.
 <pre>--name <i>vch_appliance_name</i></pre>
 
 ### `password` ###
+
+Short name: `-p`
+
 The password for the user account on the vCenter Server on which you  are deploying the virtual container host, or the password for the ESXi host if you are deploying directly to an ESXi host. If not specified, `vic-machine` prompts you to enter the password during deployment.
 
-**NOTE**: If your password contains special characters, you must wrap the password in single quotation marks (').
+**NOTE**: If your password contains special characters, you must wrap the password in single quotation marks (') on Mac OS and Linux and in double quotation (") marks on Windows.
 
 <pre>--password '<i>esxi_host_or_vcenter_server_p@ssword</i>'</pre>
 
 ### `force` ###
+
+Short name: `-f`
+
 Forces `vic-machine` to ignore warnings and non-fatal errors and continue with the deployment of a virtual container host. Errors such as an incorrect compute resource still cause the installation to fail. Using the `force` option deletes any existing virtual container host appliances or datastore folders that have the same name as the one you are specifying in the current deployment. 
```

Comment:
The delete function of force install is removed, cause we already had vic-machine delete, that one is more clear and not easy to cause unexpected result.